distribution/lib/Standard/Base/0.1.0/src/Main.enso

```diff
@@ -41,7 +41,7 @@ from Standard.Base.Data.Number.Extensions export all hiding Math, String, Double
 from Standard.Base.Data.Noise export all hiding Noise
 from Standard.Base.Data.Pair export Pair
 from Standard.Base.Data.Range export Range
-from Standard.Base.Data.Text.Extensions export Text, Split_Kind
+from Standard.Base.Data.Text.Extensions export Text, Split_Kind, Line_Ending_Style
```

Comment:
Do we really want this exported by default rather than accessible as `Text.Line_Ending_Style`?

Reply:
`Line_Ending_Style` seems rather clear to me on its own. And `Text.Line_Ending_Style` is a looot to type.

Reply:
Then I'd go with `Text.Line_Ending`. What if we ever introduce another kind of line ending style?